tests/Request/BaseRequestTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Request;
+
+use GuzzleHttp\Psr7\Request;
+use PodPoint\Reviews\Exceptions\ValidationException;
+use PodPoint\Reviews\Request\BaseRequest;
+use PodPoint\Reviews\Tests\TestCase;
+
+/**
+ * Class BaseRequestTest
+ * @package PodPoint\Reviews\Tests\Request
+ */
+class BaseRequestTest extends TestCase
+{
+
+    protected $request;
+    protected $mockedApiClient;
+
+    protected function setUp()
+    {
+        $this->mockedApiClient = $this->getMockedApiClient();
+
+        $this->request = new class($this->mockedApiClient, ['foo-required' => 'bar']) extends BaseRequest {
+
+            public function requiredFields(): array
+            {
+                return ['foo-required'];
+            }
+
+            public function getRequest(): Request
+            {
+                //
```

Comment:
This is because of the To-Do's in the source right? Boilerplate to match the boilerplate in the source?

Reply:
Yes, because the anonymous class is extending a abstract class these methods are required.